src/test/scala/mesosphere/marathon/integration/RemoteRegionOffersIntegrationTest.scala

```diff
@@ -0,0 +1,110 @@
+package mesosphere.marathon
+package integration
+
+import java.util.UUID
+
+import mesosphere.AkkaIntegrationTest
+import mesosphere.marathon.integration.facades.ITEnrichedTask
+import mesosphere.marathon.integration.facades.MarathonFacade.extractDeploymentIds
+import mesosphere.marathon.integration.facades.MesosFacade.ITResourceStringValue
+import mesosphere.marathon.integration.setup._
+import mesosphere.marathon.raml.App
+import mesosphere.marathon.state.{ FaultDomain, PathId, Region, Zone }
+import mesosphere.marathon.state.PathId._
+import mesosphere.mesos.Constraints
+
+import scala.concurrent.duration._
+
+@IntegrationTest
+class RemoteRegionOffersIntegrationTest extends AkkaIntegrationTest with EmbeddedMarathonTest {
+
+  override lazy val mesosNumMasters = 1
+  override lazy val mesosNumSlaves = 3
+
+  val homeRegion = Region("home_region")
+  val homeZone = Zone("home_zone")
+
+  val remoteRegion = Region("remote_region")
+  val remoteZone1 = Zone("remote_zone1")
+  val remoteZone2 = Zone("remote_zone2")
+
+  override def mastersFaultDomains = Seq(Some(FaultDomain(region = homeRegion, zone = homeZone)))
+
+  override def agentsFaultDomains = Seq(
+    Some(FaultDomain(region = remoteRegion, zone = remoteZone1)),
+    Some(FaultDomain(region = remoteRegion, zone = remoteZone2)),
+    Some(FaultDomain(region = homeRegion, zone = homeZone)))
+
+  before(cleanUp())
```

Comment:
why do you need to `cleanUp` between tests? We try to avoid that and make every test a _snow flake_ independent of the rest of the cluster. Ideally `cleanUp` should only be called once at the end of the test.